config.xml

```diff
@@ -289,4 +289,5 @@
             <path event="openRoasterBeans" url="/app/roaster/bean*" />
         </host>
     </universal-links>
+    <allow-navigation href="http://localhost:8100" sessionid="5ad64ca3" />
```

Comment:
Same here, is this needed? Else I'd like to revert it, localhost is always problem maker on android/ios

Reply:
Sure, I think it was added because of `ionic serve`